oneandone/src/test/java/org/apache/jclouds/oneandone/rest/features/LoadBalancerApiLiveTest.java

```diff
@@ -0,0 +1,211 @@
+public class LoadBalancerApiLiveTest extends BaseOneAndOneLiveTest {
+
+   private LoadBalancer currentLoadBalancer;
+   private Server currentServer;
+   private List<LoadBalancer> loadBalancers;
+
+   private LoadBalancerApi loadBalancerApi() {
+      return api.loadBalancerApi();
+   }
+
+   @BeforeClass
+   public void setupTest() {
+      currentServer = createServer("loadbalancers jclouds server");
+      assertNodeAvailable(currentServer);
+
+      List<LoadBalancer.Rule.CreatePayload> rules = new ArrayList<LoadBalancer.Rule.CreatePayload>();
+      LoadBalancer.Rule.CreatePayload rule = LoadBalancer.Rule.CreatePayload.builder()
+              .portBalancer(4567)
+              .portServer(4567)
+              .protocol(Types.RuleProtocol.TCP)
+              .source("0.0.0.0")
+              .build();
+      rules.add(rule);
+      currentLoadBalancer = loadBalancerApi().create(LoadBalancer.CreateLoadBalancer.builder()
+              .name("jclouds loadbalancer")
+              .rules(rules)
+              .description("description")
+              .healthCheckInterval(1)
+              .healthCheckPath("path")
+              .healthCheckTest(Types.HealthCheckTestTypes.TCP)
+              .method(Types.LoadBalancerMethod.ROUND_ROBIN)
+              .persistence(Boolean.TRUE)
+              .persistenceTime(200)
+              .build());
+   }
+
+   @Test
+   public void testList() {
+      loadBalancers = loadBalancerApi().list();
+
+      assertNotNull(loadBalancers);
+      assertFalse(loadBalancers.isEmpty());
+      Assert.assertTrue(loadBalancers.size() > 0);
+   }
+
+   public void testListWithOption() {
+      GenericQueryOptions options = new GenericQueryOptions();
+      options.options(0, 0, null, "jclouds", null);
+      List<LoadBalancer> resultWithQuery = loadBalancerApi().list(options);
+
+      assertNotNull(resultWithQuery);
+      assertFalse(resultWithQuery.isEmpty());
+      Assert.assertTrue(resultWithQuery.size() > 0);
+   }
+
+   public void testGet() {
+      LoadBalancer result = loadBalancerApi().get(currentLoadBalancer.id());
+
+      assertNotNull(result);
+      assertEquals(result.id(), currentLoadBalancer.id());
+   }
+
+   @Test(dependsOnMethods = "testGet")
+   public void testUpdate() throws InterruptedException {
+      String updatedName = "UpdatedjcloudsPolicy";
+
+      LoadBalancer updateResult = loadBalancerApi().update(currentLoadBalancer.id(), LoadBalancer.UpdateLoadBalancer.builder()
+              .name(updatedName)
+              .description("description")
+              .healthCheckInterval(40)
+              .healthCheckTest(Types.HealthCheckTestTypes.TCP)
+              .method(Types.LoadBalancerMethod.ROUND_ROBIN)
+              .persistence(Boolean.TRUE)
+              .persistenceTime(1200)
+              .build());
+
+      assertNotNull(updateResult);
+      assertEquals(updateResult.name(), updatedName);
+
+   }
+
+   @Test(dependsOnMethods = "testUpdate")
```

Comment:
Could it just depend on the get test too, top maximize the chances it gets executed if other tests fail?